cmd/kubeadm/app/constants/constants.go

```diff
@@ -398,6 +398,17 @@ const (
 
 	// CgroupDriverSystemd holds the systemd driver type
 	CgroupDriverSystemd = "systemd"
+
+	// The username of the user that kube-controller-manager runs as.
+	KubeControllerManagerUserName string = "kubeadm-kcm"
+	// The username of the user that kube-apiserver runs as.
+	KubeAPIServerUserName string = "kubeadm-kas"
+	// The username of the user that kube-scheduler runs.
```

Comment:
```suggestion
	// The username of the user that kube-scheduler runs as.
```

to be consistent with the rest

Reply:
Done. Thanks for catching my silly mistake.